src/config.c

```diff
@@ -251,10 +251,6 @@ bool config_file_parse(char* filename, PCONFIGURATION config) {
         config->address = value;
       } else if (strcmp(key, "sops") == 0) {
         config->sops = strcmp("true", value) == 0;
-      } else if (strcmp(key, "localaudio") == 0) {
-        config->localaudio = strcmp("true", value) == 0;
-      } else if (strcmp(key, "quitappafter") == 0) {
-        config->quitappafter = strcmp("true", value) == 0;
```

Comment:
I think removing these line would break configurations which has one of the following lines in their configuration files
```
localaudio = false
quitafterapp = false
```

Reply:
No, it doesn't break. I specifically tested that situation see at the end of this comment. As I noted in the commit for that change (6c8b3bb), these keys are already processed correctly in the final `else`-block a bit more down:
```c
      } else {
        for (int i=0;long_options[i].name != NULL;i++) {
          if (strcmp(long_options[i].name, key) == 0) {
            if (long_options[i].has_arg == required_argument)
              parse_argument(long_options[i].val, value, config);
            else if (strcmp("true", value) == 0)
              parse_argument(long_options[i].val, NULL, config);
          }
        }
      }
```

Boolean config-file keys are processed by the further inner `else`-block in that block, i.e. when `long_options[i].has_arg != required_argument` and when the key defaults to `false` (which both `localaudio` and `quitappafter` do).

The boolean config-file key `sops` requires a special case because the command-line argument name (used by `parse_argument` to set the config struct) is `nosops` (negated).

When I added the `quitappafter` option I didn't notice the redundancy because I was not very familiar with the code, so I just followed the example of `localaudio`. However both these `if`-blocks are actually redundant as I explained.

---

To test I simply printed the config after parsing in `main()`.

With no config file (internal defaults):
```
$ ./moonlight
quitappafter = 0
localaudio = 0
```

With config file where both are set to `false`:
```
$ ./moonlight -config test.conf
quitappafter = 0
localaudio = 0
```

With config file where both are set to `true`:
```
$ ./moonlight -config test.conf
quitappafter = 1
localaudio = 1
```